docs/documentation.md

```diff
@@ -1,275 +1,86 @@
-> of funds to specific areas is not possible. If discrimination is
-> possible, data for mul- tiple project locations (such as protected
-> areas) and their financial allocation have to be collected. KfW highly
-> encourages increasing geographical discrimination of financial flows
-> to specific project locations to the maximum possible, as this affords
-> greater aid transparency and financial accountability.
-
-##### Difference between exact 34 and approximate locations
-
-> For each project location there will be one exact or approximate
-> geolocation data set. The exact geolocation designates the
-> geographical end point of an international devel- opment assistance
-> financial flow. Approximate locations can be used however when no
-> exact geoinformation is (yet) available, or such is not be collected
-> for other reasons (details see below).
->
-> For exact locations, geographical data are to be collected with GPS
-> tools and eventually edited in a Geographical Information System
-> (GIS). Approximate locations should be given on administrative scales
-> using the GADM database as a global standard.
->
-> 34 in the sense of the IATI- Standard
+**A project location can consist of one or more features that are part of a financially supported activity where no further geographical discrimination regarding funding is possible.** An example of a single-feature project location is a financially supported hospital in a specific location. An example of a project location with multiple features is a group of adjacent agricultural plots under a financed irrigation plan that benefits all producers in that area. Another example is a set of protected areas jointly financed from a fund if, and only if, further geographical discrimination as to the allocation of funds to specific areas is not possible. If discrimination is possible, data for multiple project locations (such as protected areas) and their financial allocation have to be collected. KfW highly encourages increasing geographical discrimination of financial flows to specific project locations to the maximum possible, as this affords greater aid transparency and financial accountability.
+
+##### Difference between exact[^1] and approximate locations
+
+For each project location there will be one exact or approximate geolocation data set. The exact geolocation designates the geographical end point of an international development assistance financial flow. Approximate locations can be used however, when no exact geoinformation is (yet) available, or such is not collected for other reasons (details see below).
+
+For exact locations, geographical data are to be collected with GPS tools and eventually edited in a Geographical Information System (GIS). Approximate locations should be given on administrative scales using the GADM database as a global standard.
+
+[^1]: in the sense of the IATI- Standard
 
 ##### When to collect project location information
```

Comment:
The last title, where I will sugget this specific change on headline level... Same applies then everywhere below.